libs/net/dal/Services/ContentService.cs

```diff
@@ -210,47 +211,42 @@ public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindFr
     /// <returns>A page of content items that match the filter.</returns>
     public async Task<IPaged<API.Areas.Services.Models.Content.ContentModel>> FindWithElasticsearchAsync(string index, ContentFilter filter)
     {
-        var sourceQueries = new List<Func<QueryContainerDescriptor<API.Areas.Services.Models.Content.ContentModel>, QueryContainer>>();
-        foreach (var sourceId in filter.SourceIds)
-            sourceQueries.Add(s => s.Term(t => t.SourceId, sourceId));
+        var contentQueries = new List<Func<QueryContainerDescriptor<API.Areas.Services.Models.Content.ContentModel>, QueryContainer>>();
```

Comment:
Fixed a number of elasticsearch query issues when more than one filter value was present.